projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.coponent.ts

```diff
@@ -36,4 +36,7 @@ export class BadgeDocsComponent {
   `;
 
   pageTitle: string = 'Badge';
+  linkToSource(): void {
+    window.open('https://github.com/mobi/goponents/tree/dev/projects/go-lib/src/lib/components/go-badge','_blank')
+  }
```

Comment:
@Ashygoyal what do you think about putting these links right in the html so that it behaves as a normal link? That way we don't have to edit these typescript files.

```
<a
  href="https://github.com/mobi/goponents/tree/dev/projects/go-lib/src/lib/components/go-badge"
  target="_blank">
  <go-button buttonVariant="neutral"> Link To Source </go-button>
</a>
```

Reply:
Yeah sure, I can change these if that makes things easier.

Reply:
Okay, I see what you mean. I still think we should adopt the `<a>` approach over a click event on the button for usability, but perhaps we can go with this approach?
```ts
pageTitle: string = 'Badge';
linkToSource: string = 'https://github.com/mobi/goponents/tree/dev/projects/go-lib/src/lib/components/go-badge'
```
```html
<a
  [href]="linkToSource"
  target="_blank">
  <go-button buttonVariant="neutral"> Link To Source </go-button>
</a>
```

Reply:
@grahamhency : Sure, just implemented this change at all places where the function was getting called on button click.